rancher/terraform-provider-rancher2
# default dynamic to false

## Issue: 
https://github.com/rancher/terraform-provider-rancher2/issues/1426
 
## Solution
I removed default `true`for the `dynamic` field to align with the default API settings, which also has the added benefit of resolving the issue of duplicate entries being added.
 
### Manual Testing
- Set up an RKE2 cluster following the steps in the linked issue. Ran multiple updates on the cluster to confirm that no duplicate entries were added. 
- Changed the default PSACT from `rancher-restricted` to `rancher-privileged` and made sure it worked without causing the cluster to error out with the message `secret xxxx-psact does not contain the expected content`.
